config/_default/menus.yaml

```
@@ -116,11 +123,14 @@ community_links:
   - name: "About Events"
     weight: 3
     url: "/about-events"
-  - name: "Working Groups"
+  - name: "Organizing Events"
```

```suggestion
  - name: "Organizing NWB Events"
```

consistency across menus